components/teams/add-team-member-modal.tsx

```diff
@@ -82,26 +126,41 @@ export function AddTeamMembers({
         <DialogHeader className="text-start">
           <DialogTitle>Add Member</DialogTitle>
           <DialogDescription>
-            You can easily add team members.
+            Invite team members via email or share the invite link.
           </DialogDescription>
         </DialogHeader>
         <form onSubmit={handleSubmit}>
-          <Label htmlFor="domain" className="opacity-80">
+          <Label htmlFor="email" className="opacity-80">
             Email
           </Label>
           <Input
             id="email"
             placeholder="team@member.com"
-            className="mb-8 mt-1 w-full"
+            className="mb-4 mt-1 w-full"
             onChange={(e) => setEmail(e.target.value)}
           />
+          <Button type="submit" className="mb-6 w-full" disabled={loading}>
+            {loading ? "Sending invitation..." : "Send Invitation"}
+          </Button>
+        </form>
 
-          <DialogFooter>
-            <Button type="submit" className="h-9 w-full">
-              {loading ? "Sending email..." : "Add member"}
+        <div className="mb-4">
+          <Label className="opacity-80">Or share invite link</Label>
+          <Input value={inviteLink || ""} readOnly className="mt-1 w-full" />
+          <div className="mt-2 flex space-x-2">
+            <CopyInviteLinkButton
+              inviteLink={inviteLink}
+              className="flex-1"
+            />
+            <Button
+              onClick={handleResetInviteLink}
+              disabled={inviteLinkLoading}
+              className="flex-1"
+            >
+              Reset Link
             </Button>
-          </DialogFooter>
-        </form>
+          </div>
```

Comment:
as mentioned before, please add a separate modal for this invite link . it should not appear on the same modal as the team member invite.